examples/go/google-cloudrun/cdktf.json

```diff
@@ -0,0 +1,13 @@
+{
+  "language": "go",
+  "app": "go run main.go",
+  "codeMakerOutput": "generated",
+  "projectId": "1259b194-7cf9-49c0-aeb2-5e537d3c06da",
```

Comment:
We usually remove the project ids from our examples to leave users the choice of whether they'd like to opt into telemetry or not :)

```suggestion
```